### PR TITLE
Use pathType Prefix not Exact for ingresses

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: prow/cluster/deck.yaml [hunk 1]
```diff
@@ -107,7 +107,7 @@ spec:
       http:
         paths:
           - path: /
-            pathType: Exact
+            pathType: Prefix
```
Comment (rossf7) [inline prow/cluster/deck.yaml:110]: With this the JS links are still working for me. There is also Mixed but I think this is what we want,

https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types

**Review by MarcelMue — APPROVED** [review]
LGTM if it works. Still super confusing to me :D

File: tekton/dashboard/tekton-ingresses.yaml [hunk 2]
```diff
@@ -22,6 +22,7 @@ spec:
       http:
         paths:
           - path: /
+            pathType: Prefix
```
Comment (njuettner) [inline tekton/dashboard/tekton-ingresses.yaml:25]: I did remove it and I just saw it broke that's why I dropped it. If it's working with Prefix let's merge it :)

**Review by njuettner — APPROVED** [review]

File: tekton/dashboard/tekton-ingresses.yaml (hunk 2 above)
Comment (rossf7) [reply tekton/dashboard/tekton-ingresses.yaml:25]: @njuettner I think it was already broken. I also upgraded the Nginx IC app as there was a problem with the validation webhook.

**Pull request merged**